### DISCUSSION TITLE
Adding OW2 information on software

### DISCUSSION THREAD

**Discussion by coudot** [codegouvfr/sill-web#467] [Ideas]
Hello,

I am a member of OW2 board (https://www.ow2.org/view/About/OW2_Governance#boardmembers) and we are working on spreading the OW2 name.

Some of softwares referenced in SILL are OW2 projects and we would like to add a link to the OW2 project page in SILL, like you have for "Comptoir du Libre" for example.

Would this fit in your roadmap?

**Comment by bzg** [comment]
Hi @coudot! Sorry for the late reply.

The link to "Comptoir du libre" was done thanks to a partnership DINUM/ADULLACT: we display this link and ADULLACT provides us with information regarding FLOSS providers, as displayed in the dedicated tab "Prestataire de service".

We can explore two routes: either (1) treat OW2 as a new external source of information or (2) display the author/organisation in the `Apropos` or `Metadata` sections.

(2) coud be considered now, I'll let the devs decide.

(1) can be considered when we're done with the current iteration, implementing correct handling of multiple data endpoints for the same software.
